Lib/test/test_capi/test_opt.py

```diff
@@ -578,6 +579,13 @@ def testfunc(n):
 @requires_specialization
 @unittest.skipIf(os.getenv("PYTHON_UOPS_OPTIMIZE") == "0", "Needs uop optimizer to run.")
 class TestUopsOptimization(unittest.TestCase):
+    FRAMESIZE_ADJUSTMENT = 1 if platform.architecture()[0] == "32bit" else 0
+
+    def _assert_framesize_cross_platform(self, fn_obj, expected_framesize):
+        self.assertEqual(
+            _testinternalcapi.get_co_framesize(fn_obj.__code__),
+            expected_framesize + self.FRAMESIZE_ADJUSTMENT
```

Comment:
It isn't clear why this adjustment is necessary.
Also, would is cleaner to do the adjustment in `_testinternalcapi.get_co_framesize`?

Reply:
It isn't clear to me either why framesizes vary across platforms 😅 but based on other comments I'm thinking that this assertion is overly specific and a bit brittle, so I'm just going to remove it. I don't think I want to adjust `get_co_framesize`, since I need this to report the actual framesize in every case